bin/citgm-all.js

```diff
@@ -56,7 +65,9 @@ var options = {
   level: app.verbose,
   npmLevel: app.npmLoglevel,
   timeoutLength: app.timeout,
-  tmpDir: app.tmpDir
+  tmpDir: app.tmpDir,
+  withTags: app.withTags,
+  excludeTags: app.excludeTags
```

Comment:
if you see my above comment we can use the yargs [array key](https://www.npmjs.com/package/yargs#arraykey) here to simplify things